utils/logger/logger_test.go

```diff
@@ -1,281 +0,0 @@
-// this is a new logger interface for mattermost
-
-package logger
-
-type LogMessage struct {
-	Context map[string]string
-	File    string
-	Message string
-}
-
-const validPath = `^utils/([a-z_]+/)*logger_test.go$`
-
-/*
-
-Broken because code somehow depends on path
```

Comment:
If you commented this out thinking you would restore it eventually, I would suggest deleting it instead and giving yourself a task to restore it. In general, people are far more likely to fix broken tests when they do that, and commented out code never belongs in the repo if you ask me (All past revisions of your code are already forever accessible through Git.).

Reply:
Thanks for cleaning it up. I haven't had time to look at this code in awhile, but I'll be sure to improve the tests on it when I get a chance to get back to it.